src/main/scala/com/cognite/sdk/scala/v1/fdm/common/properties/PropertyDefinition.scala

```diff
@@ -42,34 +47,65 @@ object PropertyDefinition {
       containerPropertyIdentifier: Option[String] = None
   ) extends ViewPropertyDefinition
       with CorePropertyDefinition
+      with ViewPropertyCreateDefinition
+
+  sealed trait ConnectionDefinition extends ViewPropertyDefinition
 
-  final case class ConnectionDefinition(
+  final case class ReverseDirectRelationConnection(
+      name: Option[String],
+      description: Option[String],
+      connectionType: String,
```

Comment:
should this value be validated against https://api-docs.cognite.com/20230101/tag/Views/operation/ApplyViews?#!path=items/properties/property-identifier*/1/1/connectionType&t=request ? Looks like currrently it isn't used to distinguish rdr from others as `through` for example is required, but it seems that in spirit `connectionType` is meant as discriminator in the API

Reply:
Oh yeah, this should be an enum!

Reply:
One annoying thing is for edge connection it is marked optional, but still should be possible to limit values. Not entirely sure if scala enum would fit best here or there's some other way to put a "constant" that distinguishes the cases, but whatever works should be fine :+1:

Reply:
I made them into enum!

Reply:
By that I mean the scala equivalent with sealed abstract class, similar to what is done for direction in edgeConnections